azkaban-common/build.gradle

```diff
@@ -1,10 +1,5 @@
 configurations {
-  all {
-    transitive = false
-  }
-  errorprone {
-    transitive = true
-  }
+  errorprone
```

Comment:
What happens if this line is removed? Will this plugin use the same default config?